tests/codeception/acceptance/Install/Content/PrivatePageContentCest.php

```diff
@@ -1,30 +0,0 @@
-<?php
```

Comment:
`PrivatepageContentCest` was moved to `tests/codeception/functional/Install/Content/PrivatePageContentCest.php`, again due to the modal and component load times increasing with the new admin theme.